test/classes/delete-free/isClassTest.good

```diff
@@ -1,4 +1,4 @@
-C
+borrowed C
```

Comment:
Seems like this should be `owned C` with https://github.com/chapel-lang/chapel/pull/12916, but I'm just merging this to quiet testing, I'll let @mppf look into that.

Reply:
I think `borrowed C` is appropriate given the current language definition (#12916 only applies in the context of `new` expressions, not other cases where undecorated `C` appears.  Issue #12917 is currently wrestling with the question of what that should mean in other contexts.